generators/server/templates/src/main/java/package/config/ApplicationProperties.java.ejs

```diff
@@ -28,5 +28,21 @@ import org.springframework.boot.context.properties.ConfigurationProperties;
  */
 @ConfigurationProperties(prefix = "application", ignoreUnknownFields = false)
 public class ApplicationProperties {
+    private final AuditEvents auditEvents = new AuditEvents();
 
+    public AuditEvents getAuditEvents() {
+        return auditEvents;
+    };
+
+    public static class AuditEvents {
+        private int deletionFrequency;
```

Comment:
Won't it be better to include frequency unit like `deletionFrequencyInDays` which is more understandable by readers?

Reply:
Thanks; good idea; will make that change 😄

Reply:
Done 😄